src/components/editor/editor-component.ts

```diff
@@ -1,6 +1,7 @@
 import * as html from '@hyperapp/html'
 
 import {loadNote} from '@lib/io'
+import {writeNote} from '@lib/io'
```

Comment:
You dont need a separate import statement when importing from a single source, just group it with the `loadNote` import on previous line.

Reply:
done